test/acceptance/user_searches_announcements_test.exs

```diff
@@ -1,5 +1,5 @@
 defmodule ConstableWeb.UserSearchesAnnouncementsTest do
-  use ConstableWeb.AcceptanceCase, async: true
```

Comment:
Were the specs failing? Is that why we're removing the `async: true` here?

Reply:
Yeah, it fails while there and I don't understand why. All the docs I can find about this instruct me to do exactly what we're doing re: async working ... so I'm not sure what to try next, and just removed it. I'd love to understand what's happening and leave it.